docs/devices.rst

```diff
@@ -60,6 +60,42 @@ type of device. The available iterators are :py:class:`DeviceTokenList` and :py:
    :members: DeviceTokenList, APIDList, DeviceInfo
    :noindex:
 
+
+Open Channel Registration
+--------------
+
+Open Channels are registered by using :py:class:`OpenChannel`.
+
+.. code-block:: python
+
+   import urbanairship as ua
+   airship = ua.Airship(app_key, app_secret)
+
+   my_channel = ua.OpenChannel()
+   my_channel.address = 'my_email@example.com'
+   my_channel.open_platform = 'email'
+   my_channel.opt_in = True
+   my_channel.create(airship)
```

Comment:
This took me a while to deduce, but it seems that the hydration is subject to race condition. I was fetching the full channel payload after creation to populate extra fields such as the created and last modified date, but sometimes the channel creation hasn't populated to the GET endpoint yet, and so it will 404. I will just remove the hydration piece at the end. I am glad you caught this, thank you!